products/PurePhone/services/appmgr/CMakeLists.txt

```diff
@@ -3,6 +3,7 @@ add_library(appmgr STATIC)
 target_sources(appmgr
     PRIVATE
         ApplicationManager.cpp
+        $<$<BOOL:${ENABLE_APP_SPECIAL_INPUT}>:RunAppsInBackground.cpp>
```

Comment:
I know what you are trying to achieve, but IMHO this looks strange ;) It only shows we are snowed under unclear dependencies that lead to strange constructs.

Reply:
True (https://github.com/mudita/MuditaOS/pull/3648#discussion_r828913203) there is no better way at the moment :(